### DISCUSSION TITLE
suggestions

### DISCUSSION THREAD

**Discussion by Dan4oby** [ACGaming/UniversalTweaks#347] [Tweak Suggestions]
add some fixes from here as it seems to be abandoned

https://github.com/mrgrim/MUP

**Comment by Dan4oby** [comment]
also

https://www.curseforge.com/minecraft/mc-mods/randompatches-forge

**Comment by ACGaming** [comment]
MUP is a great source since mrgrim did a good job with many tweaks, I already included some of them. Random Patches is on the list as well.

**Comment by Dan4oby** [comment]
> MUP is a great source since mrgrim did a good job with many tweaks, I already included some of them. Random Patches is on the list as well.

thats good 
also you can integrate something from these
https://modrinth.com/mod/sledgehammer
https://modrinth.com/mod/darkredstone
https://www.curseforge.com/minecraft/mc-mods/born-in-a-barn
https://www.curseforge.com/minecraft/mc-mods/netherportalfix

**Comment by Turkey9002** [comment]
The licenses of Born in a Barn and NetherPortalFix won't allow them to be integrated just to let you know.

**Comment by Krutoy242** [comment]
Suggestion from me.  
This mostly small mods that adds one or two fix/feature. I don't know if they have appropriate license or actually worst to add into UT.

- [x] [SwingThroughGrass](https://www.curseforge.com/minecraft/mc-mods/swingthroughgrass)
- [x] [Ding](https://www.curseforge.com/minecraft/mc-mods/ding)
- [ ] [BetterFPS](https://www.curseforge.com/minecraft/mc-mods/betterfps)
- [x] [Enable Cheats](https://www.curseforge.com/minecraft/mc-mods/enable-cheats)
- [x] [Smooth Scrolling Everywhere](https://www.curseforge.com/minecraft/mc-mods/smooth-scrolling-everywhere)
- [x] [Stupid Horse Stand Still](https://www.curseforge.com/minecraft/mc-mods/stupid-horse-stand-still)
- [ ] [AI Reducer](https://www.curseforge.com/minecraft/mc-mods/ai-reducer)
- [ ] [Better Biome Blend](https://www.curseforge.com/minecraft/mc-mods/better-biome-blend)

**Comment by ACGaming** [comment]
Just a quick heads-up, there‘s a master list for all requested mod features now where you can see which are already on my radar along with their current state: https://github.com/ACGaming/UniversalTweaks/wiki/Requested-Features

**Comment by jbredwards** [comment]
https://www.curseforge.com/minecraft/mc-mods/end-portal-parallax
https://www.curseforge.com/minecraft/mc-mods/elytra-aeronautics
https://www.curseforge.com/minecraft/mc-mods/ferritecore
https://www.curseforge.com/minecraft/mc-mods/item-highlighter (1.16.5 is MIT)
https://www.curseforge.com/minecraft/mc-mods/loot-beams
https://www.curseforge.com/minecraft/mc-mods/sound-physics-remastered

also it might be a good idea to enable this repo's discussions tab for feature requests?

**Comment by ACGaming** [comment]
This is now a thing: https://github.com/ACGaming/UniversalTweaks/discussions

**Comment by Turkey9002** [comment]
It be great to also organize different categories depending on the type of request (tweak or bug fix), as well as whether it already exists in 1.12.2 or if it comes from a version prior (port) or after (backport). 

And dang Mojang really did the end portal dirty after 1.11...

**Comment by ACGaming** [comment]
Added the categories to the discussion tab.
